common/static/common/js/jasmine_stack_trace.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+    /* globals jasmineRequire */
+    'use strict';
+
+    var OldExceptionFormatter = jasmineRequire.ExceptionFormatter(),
+        oldExceptionFormatter = new OldExceptionFormatter();
+
+    jasmineRequire.ExceptionFormatter = function () {
+        function ExceptionFormatter() {
+            this.message = oldExceptionFormatter.message;
+            this.stack = function (error) {
+                var errorMsg = null;
+
+                if (error) {
+                    errorMsg = error.stack.split('\n').slice(0, 10).join('\n');
```

Comment:
Make a variable for the number 10 so it is clearer to someone reading this what is happening. Something like ``MAX_STACK_TRACE_LINES``.